test/watchman_heartbeat_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule WatchmanHeartbeatTest do
```

Comment:
This file should be in `test/lib/watchman/heartbeat_test.exs`. This will enable us to quickly jump between test and implantation in Vim.